packages/marketing-modal/src/Graphics/graphics.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
I think this file name should be capitalized (correct me if I'm wrong)

Reply:
oh good catch